test/integration/integration_test.py

```diff
@@ -0,0 +1,85 @@
+from contextlib import contextmanager
+from marquez_client.marquez import MarquezClient
+from marquez_codegen_client.rest import ApiException
+from time import sleep
+from urllib3.exceptions import (MaxRetryError,
+                                NewConnectionError,
+                                ReadTimeoutError)
+
+import pytest
+import requests
+import subprocess
+import sys
+
+MARQUEZ_HOST = "marquez"
+MARQUEZ_PORT = "5000"
+
+
+def test_bad_host():
+    assert wait_for_marquez()
```

Comment:
If we do this for each method in this class, we can just create a test fixture for it.

Reply:
That test fixture can be scoped to 'class', so that it runs exactly once.

Reply:
sounds good